src/patternfly/components/Menu/menu.scss

```diff
@@ -489,298 +385,217 @@
-    margin-inline-start: var(--#{$menu}__header--c-menu__item--MarginLeft);
-    margin-inline-end: var(--#{$menu}__header--c-menu__item--MarginRight);
-
-    &:hover {
-      --#{$menu}__item--BackgroundColor: var(--#{$menu}__header--c-menu__item--hover--BackgroundColor);
-    }
-
-    &:focus {
-      --#{$menu}__item--BackgroundColor: var(--#{$menu}__header--c-menu__item--focus--BackgroundColor);
-    }
-  }
-
-  + .#{$menu}__search {
-    --#{$menu}__search--PaddingTop: var(--#{$menu}__header__search--PaddingTop);
-  }
-}
-
-// Search
-.#{$menu}__search {
-  padding-block-start: var(--#{$menu}__search--PaddingTop);
-  padding-block-end: var(--#{$menu}__search--PaddingBottom);
-  padding-inline-start: var(--#{$menu}__search--PaddingLeft);
-  padding-inline-end: var(--#{$menu}__search--PaddingRight);
-}
-
-// List
-.#{$menu}__list {
-  @include pf-v5-hidden-visible(var(--#{$menu}__list--Display));
-
-  padding-block-start: var(--#{$menu}__list--PaddingTop);
-  padding-block-end: var(--#{$menu}__list--PaddingBottom);
-
-  > .#{$divider} {
-    margin-block-start: var(--#{$menu}__list--c-divider--MarginTop);
-    margin-block-end: var(--#{$menu}__list--c-divider--MarginBottom);
-  }
+// - Menu list
+.#{$menu}__list :where(.#{$divider}:is(li)) {
+  margin-block-start: var(--#{$menu}__list--divider--MarginTop, var(--#{$menu}--RowGap));
+  margin-block-end: var(--#{$menu}__list--divider--MarginBottom, var(--#{$menu}--RowGap));
 }
 
-// List item
+// - Menu list item
 .#{$menu}__list-item {
-  @include pf-v5-hidden-visible(var(--#{$menu}__list-item--Display));
-
-  position: relative;
+  align-items: start;
+  min-width: 0; // allow list item to shrink for text overflow
   color: var(--#{$menu}__list-item--Color);
   background-color: var(--#{$menu}__list-item--BackgroundColor);
 
-  &:hover {
-    --#{$menu}__list-item--BackgroundColor: var(--#{$menu}__list-item--hover--BackgroundColor);
-    --#{$menu}__list-item--Color: var(--#{$menu}__list-item--hover--Color, inherit);
-  }
-
-  &:focus-within,
-  &.pf-m-focus {
-    --#{$menu}__list-item--BackgroundColor: var(--#{$menu}__list-item--focus-within--BackgroundColor);
-    --#{$menu}__list-item--Color: var(--#{$menu}__list-item--focus-within--Color, inherit);
-  }
-
-  &.pf-m-disabled,
-  &.pf-m-aria-disabled {
-    --#{$menu}__list-item--hover--BackgroundColor: transparent;
-    --#{$menu}__item--Color: var(--#{$menu}__list-item--m-disabled__item--Color);
-    --#{$menu}__item-toggle-icon: var(--#{$menu}__list-item--m-disabled__item-toggle-icon--Color);
+  &:hover .#{$menu}__list-item {
+    background-color: var(--#{$menu}__list-item--child--not-active--BackgroundColor);
   }
 
-  &.pf-m-disabled {
-    --#{$menu}__list-item--focus-within--BackgroundColor: transparent;
-
-    pointer-events: none;
-  }
+  &:has(.#{$menu}__item:hover, .#{$menu}__item:focus) {
+    --#{$menu}__list-item--Color: var(--#{$menu}__list-item--hover--Color);
```

Comment:
`--#{$menu}__list-item--hover--Color` is undefined, though I'm not sure what `--#{$menu}__list-item--Color` does. Setting it doesn't appear to change any styles in the menu as far as I could tell - do you know where it's used?